api/api/tags.py

```diff
@@ -22,3 +22,4 @@ class RouteTags(str, Enum):
     FEATURES = "Features"
     PROMPT_ENGINEER_AGENT = "Prompt Engineer Agent"
     FEEDBACK = "Feedback"
+    WEBHOOKS = "Webhooks"
```

Comment:
Can you add to the other webhooks then ?

Reply:
After thinking about it, it doesn't make sense if we don't display the webhooks in the docs.